supabase/migrations/20260721110632_admin_upgrade_rate_12m.sql

```
@@ -0,0 +1,5 @@
+ALTER TABLE public.global_stats
+  ADD COLUMN IF NOT EXISTS upgrade_rate_12m double precision NOT NULL DEFAULT 0;
+
+COMMENT ON COLUMN public.global_stats.upgrade_rate_12m
+  IS 'Percentage of organizations whose last stripe_info.upgraded_at falls within the trailing 12 calendar months ending at the UTC snapshot day end (orgs with last stripe_info.upgraded_at in-window / orgs created by day end * 100).';
```

SQLFluff (4.2.2)

[error] Line is too long (234 > 80).

(LT05)